Seminars/Sem.01/Pract.01/task04.cpp

```diff
@@ -0,0 +1,30 @@
+<iostream>
+
+using namespace std;
+
+struct Student {
+   char name[30];
+   int facultyNumber;
+};
+
+void initStudent(Student &student) {
+   cout << "Enter name: ";
+   cin.getline(student.name, 30);
+   do {
+       cout << "Enter faculty number: ";
+       cin >> student.facultyNumber;
+   } while (student.facultyNumber <= 0);
+   cin.ignore();
+}
+
+void printStudent(Student const &student) {
+   cout << "Name: " << student.name << endl;
+   cout << "Faculty number: " << student.facultyNumber << endl;
+}
+
+int main() {
+   Student student{};
```

Comment:
What do the curly brackets do in this case?
Explain.

Reply:
When we add `{}` to the declaration of the struct it value initializes all members to 0/null based on the member type. Also I've heard it's good practice.